src/__tests__/union.test.ts

```diff
@@ -101,7 +101,7 @@ describe('union', () => {
                     const ufs = new Union();
                     ufs.use(vol as any);
                     ufs.use(vol2 as any);
-                    expect(ufs.readdirSync("/foo")).toEqual(["bar", "baz", "qux"]);
+                    expect(ufs.readdirSync("/foo")).toEqual(["qux", "bar", "baz"]);
```

Comment:
Ah guess I was wrong - the `sort` needs to stay

Reply:
I've expected that ;) I remove the commit.